tests/test_bilstm.py

```diff
@@ -162,3 +162,41 @@ def test_early_stopping():
     # a lot of time to finish running this test
     model.fit(X, Y)
     assert model.score(X, Y) > 0.6
+
+
+def test_predict_proba():
+    X = [
+        "One",
+        "One only",
+        "Two nothing else",
+        "Two and three"
+    ]
+    Y = np.array([0, 0, 1, 1])
```

Comment:
Can we add a test where Y is a sparse array ? I think the tests are missing that part of the code.

Reply:
good point

Reply:
As it turns out there is a test for sparse.
https://github.com/wellcometrust/WellcomeML/blob/6b7503a043157edf5e40d22b9c2c312d255b60ae/tests/test_cnn.py#L96

Did you mean about proba? I removed sparse_y from proba as it did not make sense